### ISSUE TITLE
Datetime widget hidden input don't get the value when ajaxConversion is true 

### ISSUE THREAD

**Issue by aaron-cardenas** [kartik-v/yii2-datecontrol#79]
I have a form whit 

<?= $form->field($model, 'fecha_de_adhesion')->widget(DateControl::classname(), [
        'type'=>DateControl::FORMAT_DATETIME,  
]); ?>

If i set ajaxConversion to false all works as expected, except i can't set timezones and i get wrong values.

Whit ajaxConversion true in some specific conditions the widget stop working.

It only hapoend to datetime widget, date and time widgets works ok.

Depend on de language set in the yii configuration. Whit 'language' => 'en' or fr all works, but es or de make it fail. I didn't test more languages.

**Comment by iurijacob** [comment]
I'm having this same issue with pt-br language. Setting ajaxConversion = true the form don't get validated as it should be, returning the message that my datetime field is empty.

**Comment by Enrica-r** [comment]
@aaron-cardenas 
Javascript conversion ('ajaxConversion'=false) has no time zone conversion built in. If you have to have it you must use ajax. Datecontrol with 'datetime' works in my application with or without ajaxConversion. Please check if the displayed format really matches with the expected display format.

@iurijacob Kartik's demo has a wrong formatted datetime display field. The original value is shown without space. If you enter a value there is a space between the year and time. I don't know what his demo expected.

**Comment by iurijacob** [comment]
Neither do I...

**Comment by kartik-v** [comment]
The issue you mention above is specific to Yii Masked Input... which needs to be checked. You may try with a normal text input and try or use one of the other date/time widgets.

**Comment by kartik-v** [comment]
The demo has been fixed for Yii Masked Input configuration (the field hint will show the `displayFormat` and `mask`).

**Comment by Enrica-r** [comment]
@kartik-v 
Your demo is working for Masked input but not with JUI Datepicker (datetime and date). see
![error_datetime_demo](https://cloud.githubusercontent.com/assets/2463365/21761308/62e45c3c-d652-11e6-929f-4cd54b0327ca.JPG)

**Comment by kartik-v** [comment]
You may need to check it again and confirm,

Note that if you set `autoWidget` to true then the JUI datepicker will not be used (it will use inbuilt Krajee widgets automatically),

If you need to use JUI datepicker in the demo then set  `autoWidget` toggle to OFF (i.e. false).

**Comment by Enrica-r** [comment]
In this case it's not JUI what isn't working. See the picture in my post above. It's demo problem only. In my app Datecontrol ist working well.

**Comment by kartik-v** [comment]
You can check now... its been corrected and now the code snippet will be dynamically changed to show what should be the configuration for each selected config.
